### PR TITLE
Various fixes

### LINKED ISSUE #155: Molecular transformer wrong energy usage
## Expected Behavior
Expected it to use as much time as recipe states and that much ampreser

## Current Behavior
Atm uses 2 less time then in recipe and completly ignores amperes stated in recipe

## Game Environment
 Include as many relevant details about the game environment or modpack you experienced the bug in/on.
 If you are using a Modpack:
 * Modpack Name:  GTNH
 * Version used: 2.3gf

### LINKED ISSUE #9879: GT++ molecular transformer miss recipes
### Your GTNH Discord Username

Lewissaber#7231

### Your Pack Version

 2.1.2.3QF

### Your Server

SP

### Type of Server

Single Player

### Your Expectation

Recipes are not visible in NEI and Multiblock cant do any of the adv solar panel MT recipes

### The Reality

---

### Your Proposal

Fix it already bruh

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

### PR DESCRIPTION
**Pull request by miozune**
- Remove speed bonus from MT because it's confusing
  - Closes https://github.com/GTNewHorizons/GTplusplus/issues/155
  - Closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9879
- Fix inaccurate controller GUI texture for MT
- Change amperage to 1 for MT. Most of the multis don't consume 2 amperage from 1 energy hatch, and this multi accepts only 1 energy hatch.
- Fix hacky facing validation in MT
- Adapt almost all the multiblock controllers to new `TextureFactory`. As a result, fix rotating them not updating textures.
  - Exceptions: `GregtechMetaTileEntity_LargerTurbineBase`, `GregtechMetaTileEntity_PowerSubStationController` and `GregtechMeta_SteamMultiBase`. Changing these requires breaking change to public/protected methods/fields. Or if Alk decides to break backcompatibility, we can modify these.

### CODE REVIEW

**Comment by chochem** [comment]
did you test this? These changes look incorrect to me. Specifically it never had an EU discount.  This looks to me like you just set the eu cost to 0. 
image

eidt: all cleared up.

**Comment by miozune** [comment]
Indeed I did test it. `getEuDiscountForParallelism` is generally used only for Tricoder result. (and some of the multis use it for actually calculating energy usage, but it's not the case with MT.)
See `GregtechMetaTileEntity_IsaMill` for example.

**Comment by miozune** [comment]
Ah yes we didn't have EU discount but did have speed bonus, my initial description was wrong

File: src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialArcFurnace.java [hunk 1]
```diff
@@ -205,8 +206,11 @@ public String getSound() {
 	@Override
 	public ITexture[] getTexture(final IGregTechTileEntity aBaseMetaTileEntity, final byte aSide, final byte aFacing, final byte aColorIndex, final boolean aActive, final boolean aRedstone) {
 		if (aSide == aFacing) {
+			if (aActive)
+				return new ITexture[]{Textures.BlockIcons.getCasingTextureForId(mCasingTextureID),
```
Comment (repo-alt) [inline src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialArcFurnace.java]: After the fixes this logic became big enough (and repeated in all the multies) to be extracted in a base class method perhaps?

**Review by repo-alt — APPROVED** [review]

**Pull request merged**